ios/Podfile

```diff
@@ -8,6 +8,8 @@ target 'mapeo' do
 
   use_react_native!(:path => config["reactNativePath"])
 
+  pod 'RNCCheckbox', :path => '../node_modules/@react-native-community/checkbox'
```

Comment:
same comment about autolinking taking care of this automatically. don't think this is needed